DachiCharkviani/openlibrary
# implemented the Richer UI access when downloading the app

PR Template
What issue does this PR close?

Closes N/A

What does this PR achieve?

Feature: Adds richer install UI for Progressive Web App (PWA) by incorporating screenshots for both desktop and mobile, along with a description.

Technical
This PR updates the manifest file to include screenshots for different devices (desktop and mobile). It utilizes the form_factor property to differentiate between wide (desktop) and narrow (mobile) screens, ensuring compatibility with Chrome's richer install UI.

Screenshots were added for the following contexts:

Desktop views of key pages like Main Page, Search Books, and Student Library.
Mobile views of the same pages optimized for narrow displays.
The manifest.json file was updated, and screenshots were uploaded to [location of screenshots, e.g., a public CDN or repository path].

Testing
To verify:

Clone the repository and switch to this branch.
Link the updated manifest.json to your development build.
Use Chrome to inspect the richer install UI for PWAs:
Open chrome://inspect/#apps.
Check that all screenshots are visible for both desktop and mobile contexts.
Test the UI responsiveness by viewing the install banner on different screen sizes.
Screenshot
Evidence of functionality:

Desktop screenshots:
Mobile screenshots:
Stakeholders
@RayBB (Lead)
[Other stakeholders as identified]
